.licenserc.yaml

```diff
@@ -83,3 +83,18 @@ header: # `header` section is configurations for source codes license header.
 dependency:
   files:
     - go.mod
+
+language:
+  Go:
+    type: programming
+    color: "#00ADD8"
+    aliases:
+      - golang
+    extensions:
+      - ".go"
+    tm_scope: source.go
+    ace_mode: golang
+    codemirror_mode: go
+    codemirror_mime_type: text/x-go
+    language_id: 132
+    comment_style_id: SlashAsterisk
```

Comment:
Please minimize the configs, we only need some of the fields, like `comment_style_id` and `extensions`,

Reply:
> Please minimize the configs, we only need some of the fields, like `comment_style_id` and `extensions`,

Hi @kezhenxu94 , Thanks for reply, I will fix it.